src/main/java/com/igorbochkor/dao/impl/ManufacturerDaoJdbcImpl.java

```diff
@@ -0,0 +1,121 @@
+package com.igorbochkor.dao.impl;
+
+import com.igorbochkor.dao.ManufacturerDao;
+import com.igorbochkor.exception.DataProcessingException;
+import com.igorbochkor.lib.Dao;
+import com.igorbochkor.model.Manufacturer;
+import com.igorbochkor.util.ConnectionUtil;
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+@Dao
+public class ManufacturerDaoJdbcImpl implements ManufacturerDao {
+
+    @Override
+    public Manufacturer create(Manufacturer manufacturer) {
+        String query = "INSERT INTO manufactures (name, country) VALUE (?, ?)";
+        try (Connection connection = ConnectionUtil.getConnection()) {
+            PreparedStatement statement
+                    = connection.prepareStatement(query, Statement.RETURN_GENERATED_KEYS);
+            statement.setString(1, manufacturer.getName());
+            statement.setString(2, manufacturer.getCountry());
+            statement.executeUpdate();
+            ResultSet resultSet = statement.getGeneratedKeys();
+            if (resultSet.next()) {
+                manufacturer.setId(resultSet.getLong(1));
```

Comment:
Be attentive with:
      Wrong:
          resultSet.getLong("manufacturer_id"); // return '0' if data is absent.
      Good:
           resultSet.getObject("manufacturer_id", Long.class); // return 'null' if data is absent.
          (Common mistakes)

Reply:
done